modules/installation-user-infra-machines-static-network.adoc

```diff
@@ -271,6 +271,67 @@ bond=bond0:em1,em2:mode=active-backup
 vlan=bond0.100:bond0
 ----
 
+[id="bonding_multiple_sriov_network_interfaces_to_dual_port_{context}"]
+[discrete]
+=== Bonding multiple SR-IOV network interfaces to a dual port NIC interface
+
+[IMPORTANT]
+====
+Support for Day 1 operations associated with enabling NIC partitioning for SR-IOV devices is a Technology Preview feature. Technology Preview features are not supported with Red Hat production service level agreements (SLAs) and might not be functionally complete. Red Hat does not recommend using them in production. These features provide early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process.
+
+For more information about the support scope of Red Hat Technology Preview
+features, see https://access.redhat.com/support/offerings/techpreview/.
+====
+
+Optional: You can bond multiple SR-IOV network interfaces to a dual port NIC interface by using the `bond=` option.
+
+.Prerequisties
```

Comment:
This is usually for a procedure module, but this isn't that. I'd recommend writing it out as a lead-in sentence instead.